tests/topotests/bgp_vrf_dynamic_route_leak/test_bgp_vrf_dynamic_route_leak_topo1.py

```diff
@@ -0,0 +1,1856 @@
+            input_dict_isr.update(temp)
+
+            temp[dut]["bgp"].append(
+                {
+                    "local_as": as_num,
+                    "vrf": vrf,
+                    "address_family": {
+                        addr_type: {
+                            "unicast": {
+                                "import": {
+                                    "vrf": "ISR",
+                                    "delete": True
+                                }
+                            }
+                        }
+                    }
+                })
+
+        result = create_router_bgp(tgen, topo, input_dict_isr)
+        assert result is True, "Testcase {} : Failed \n Error: {}". \
+            format(tc_name, result)
+
+        step("Verify that when import vrf ISR command is deleted, "
+             "all routes of vrf ISR disappear from default vrf and "
+             "when it's re-configured, repeat step-4 for verification.")
+
+        input_routes_r1 = {
+            "r1": {
+                "static_routes": [{
+                    "network": [
+                        NETWORK1_1[addr_type]
+                    ],
+                    "vrf": "default"
+                }]
+            }
+        }
+
+        result = verify_bgp_rib(tgen, addr_type, "r1", input_routes_r1,
+                                expected=False)
+        assert result is not True, (
+            "Testcase {} : Failed \n Routes are still present, Error {}". \
+                format(tc_name, result))
+
+        input_dict_isr["r1"]["bgp"][0]["address_family"][addr_type]["unicast"][
+            "import"]["delete"]=False
+
+        result = create_router_bgp(tgen, topo, input_dict_isr)
+        assert result is True, "Testcase {} : Failed \n Error: {}". \
+            format(tc_name, result)
+
+        result = verify_bgp_rib(tgen, addr_type, "r1", input_routes_r1)
+        assert result is True, (
+            "Testcase {} : Failed \n Error {}". \
+                format(tc_name, result))
+
+    for addr_type in ADDR_TYPES:
+
+        step("Delete and re-configure route-map IMP from global config when "
+            "import and route-maps are applied in a ISR vrf.")
+
+        input_dict_rm = {
+            "r1": {
+                "route_maps": {
+                    "rmap_IMP_{}".format(addr_type): [{
+                        "action": "permit",
+                        "match": {
+                            "community_list": {"id": "COMM"}
+                        },
+                        "set": {
+                            "community": {"num": "500:500"}
+                        },
+                        "delete": True
+                    }]
+                }
+            }
+        }
+
+        result = create_route_maps(tgen, input_dict_rm)
+        assert result is True, 'Testcase {} : Failed \n Error: {}'.format(
+            tc_name, result)
+
+        step("Waiting for 10 sec after deleteing route-map")
```

Comment:
Typo: `deleteing`

Reply:
Thanks Rafael, No longer needed as sleep is not required.